terraform/modules/hub/hub_frontend.tf

```
@@ -128,6 +132,49 @@ resource "aws_ecs_service" "frontend_v2" {
   }
 }
 
+resource "aws_ecs_task_definition" "frontend_fargate" {
+  family                   = "${var.deployment}-frontend-fargate"
+  container_definitions    = data.template_file.frontend_task_def.rendered
+  network_mode             = "awsvpc"
+  execution_role_arn       = module.frontend_ecs_roles.execution_role_arn
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = var.ingress_instance_type == "t3.xlarge" ? 2048 : 512
```

```suggestion
  cpu                      = var.ingress_instance_type == "t3.xlarge" ? 2048 : 1024
```

nginx_cpu = 256 + frontend_cpu = 768 = 1024

---

oops :+1: